erpnext/stock/doctype/day_end_stock_ledger_check/day_end_stock_ledger_check.py

```
@@ -0,0 +1,29 @@
+
+	from typing import TYPE_CHECKING
+
+	if TYPE_CHECKING:
+		from frappe.types import DF
+
+		from erpnext.stock.doctype.stock_ledger_variance.stock_ledger_variance import StockLedgerVariance
+
+		company: DF.Link | None
+		entries: DF.Table[StockLedgerVariance]
+		posting_date: DF.Date | None
+		status: DF.Literal["", "Queued", "Completed"]
```

Ruff (0.13.1)

Syntax error in forward annotation: Expected an expression

(F722)